cmd/odo/odo.go

```diff
@@ -77,7 +79,7 @@ func main() {
 		case message := <-updateInfo:
 			log.Italic(message)
 		default:
-			glog.V(4).Info("Could not get the latest release information in time. Never mind, exiting gracefully :)")
+			klog.V(4).Info("Could not get the latest release information in time. Never mind, exiting gracefully :)")
```

Comment:
why are we moving to klog instead of glog?

Reply:
klog is kubernetes fork of glog. Glog is not maintained anymore.

https://github.com/kubernetes/klog

Reply:
:+1: